README.md

```diff
@@ -193,6 +193,15 @@ It is also possible to pass the name of a migration in order to just run the mig
 umzug.up({ to: '20141101203500-task' }).then(function (migrations) {});
 ```
 
+You also have the ability to choose to run migrations *from* a specific migration, excluding it:
+
+```js
+umzug.up({ from: '20141101203500-task' }).then(function (migrations) {});
+```
+
+In the above example umzug will execute all the pending migrations found **after** the specified migration. This is particularly usefull if you are using migrations on your native desktop application and you don't need to run past migrations on new installs while they need to run on updated installations.
+
+
```

Comment:
Should we have an additional example like `umzug.up({ from: '...', to: '...' })`?